internal/audit/audit.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package audit
+
+import (
+	"crypto/md5"
+	"fmt"
+	"github.com/juju/loggo"
+	"strings"
+)
+
+var logger = loggo.GetLogger("audit")
+
+
+type Operation string
+
+const (
+	SET_READ Operation = "set_read"
+	SET_WRITE Operation = "set_write"
+)
+
+func LogAcl(entity string, operation Operation, final_state []string) {
+	state := make([]string, len(final_state))
+	for k, v := range final_state {
+		hasher := md5.New()
```

Comment:
We decided it was not a good solution for the time being, we will revisit that later if we feel the need for anonymisation on those logs.